entrypoint.sh

```diff
@@ -73,9 +73,9 @@ if expr match $1 '.*supervisord' >/dev/null; then
   if [ -f $aliases ]; then
     newaliases
   fi
-  
+
   # Configure authentification to relay if needed
-  if [ -n "${RELAY_LOGIN}" -a -n "${RELAY_PASSWORD}" ]; then
+  if [ \( -n "${RELAY_LOGIN}" -a -n "${RELAY_PASSWORD}" \) -o \( -f /etc/postfix/sasl_passwd \) ]; then
     postconf -e 'smtp_sasl_auth_enable = yes'
     # use password from hash database
     if [ -f /etc/postfix/sasl_passwd ]; then
```

Comment:
But according to the first "if", if the password database exists /etc/postfix/sasl_passwd the "else" below will never be reached again. We need to invert the if logic and test again for environment vars in "if" statement instead of testing the file existence

Reply:
I have inverted the logic as you suggested so that environment variables will prevail over an existing hashed database

Reply:
Thanks for your contributions (including typo in docs :) ) I've merged into develop, so I will perform some tests (and try to include automatic tests here if I can)
I keep you informed when the fix will arrived in master